genie-web/src/integTest/java/com/netflix/genie/web/apis/rest/v3/controllers/JobRestControllerIntegrationTest.java

```diff
@@ -267,12 +267,7 @@ public void testForTooManyCommandArgs() throws Exception {
 
     private void submitAndCheckJob(final int documentationId, final boolean archiveJob) throws Exception {
         Assume.assumeTrue(SystemUtils.IS_OS_UNIX);
-        final List<String> commandArgs;
-        if (this.agentExecution) {
-            commandArgs = Lists.newArrayList("-c", "echo hello world");
-        } else {
-            commandArgs = Lists.newArrayList("-c", "'echo hello world'");
-        }
+        final List<String> commandArgs = Lists.newArrayList("-c", "echo hello world");
```

Comment:
Would we prefer the tests to match how they were in the past? e.g. `-c 'echo hello world'`?

Reply:
I don't think that's possible because we lose the quotes in the process.
So we could keep the two execution modes the same without quotes, or we need split logic for the two cases and keep the quotes for the legacy path.